network-api/networkapi/settings.py

```diff
@@ -131,16 +131,17 @@
 
     'wagtail.wagtailforms' if ENABLE_WAGTAIL else None,
     'wagtail.wagtailredirects' if ENABLE_WAGTAIL else None,
-    'wagtail.wagtailembeds' if ENABLE_WAGTAIL else None,
+    'wagtail.wagtailembeds',
     'wagtail.wagtailsites' if ENABLE_WAGTAIL else None,
     'wagtail.wagtailusers',
     'wagtail.wagtailsnippets',
     'wagtail.wagtaildocs',
-    'wagtail.wagtailimages' if ENABLE_WAGTAIL else None,
+    'wagtail.wagtailimages',
     'wagtail.wagtailsearch' if ENABLE_WAGTAIL else None,
     'wagtail.wagtailadmin' if ENABLE_WAGTAIL else None,
     'wagtail.wagtailcore',
     'wagtail.contrib.modeladmin',
+    'wagtail.contrib.wagtailstyleguide' if ENABLE_WAGTAIL and DEBUG else None,
```

Comment:
a convenience app for devs, with some information which icon shorthand codes exist, etc.